README.md

```diff
@@ -174,4 +174,48 @@ struct SubCommandTwo {
 }
 ```
 
+## Attribute Summary
+### Type attributes for `argh`
+
+The attributes used to configure the argh information for a type are defined in
+[parse_attrs::TypeAttrs](argh_derive/src/parse_attrs.rs).
+
+* `subcommand` - a subcommand type. This attribute must appear on both enumeration and each struct that
+    is a variant for the enumerated subcommand.
+* `error_code(code, description)` - an error code for the command. This attribute can appear zero
+    or more times.
+* `examples=` - Formatted text containing examples of how to use this command. This
+   is an optional attribute.
+* `name=` - (required for subcommand variant) the name of the subcommand.
+* `notes=` - Formatted text containing usage notes for this command. This
+   is an optional attribute.
+    pub error_codes: Vec<(syn::LitInt, syn::LitStr)>,
+
```

Comment:
nit: Some of these lines are indented more than they should be, for example, 2 extra spaces before "is" on line 184, etc. 

Consider using VS Code "rewrap" extension (if using VS Code). It's usually pretty smart about indenting a selection based on its context.

I see similar indent issues below as well. (line 202, 204-205, etc.)

Reply:
@erickt - It looks like Clay has added some great documentation to the README. Can you please review the content to make sure it aligns with your expectations?

Reply:
Added more documentation about the JSON format and cleaned up the formatting.